src/app/pages/DashboardPage/ComputeNodes.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useTheme } from '@mui/material/styles'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import { SnapshotCard } from './SnapshotCard'
+import { COLORS } from '../../../styles/theme/colors'
+
+export const ComputeNodes: FC = () => {
+  const { t } = useTranslation()
+  const theme = useTheme()
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'))
+
+  return (
+    <SnapshotCard title={t('computeNodes.title')}>
+      <Box
+        sx={{
+          textAlign: 'center',
+          py: isMobile ? 3 : 4,
+        }}
+      >
+        <Typography component="span" sx={{ fontSize: '48px', fontWeight: 700, color: COLORS.brandDark }}>
+          {/* TODO: Replace with real value when endpoint is ready */}
+          {t('computeNodes.value', { value: 2000 })}
```

Comment:
https://www.oasisscan.com/paratimes/000000000000000000000000000000000000000000000000e2eaa99fc008f87f/nodeList says current number is: 70 active nodes, 1 inactive

Reply:
I will validate what number we want to show when API is ready. Is it a number of all or just active nodes.

Reply:
We end up with a new mockup for this section, PR updated.